diff -Nurp gmameui.orig/src/gui_prefs.c gmameui/src/gui_prefs.c
--- gmameui.orig/src/gui_prefs.c	2010-09-15 10:59:38.000000000 +0100
+++ gmameui/src/gui_prefs.c	2014-10-13 01:06:07.290499938 +0100
@@ -763,7 +763,7 @@ static void mame_gui_prefs_save_int (Mam
 
 	/* Set the value and save the file */
 	g_key_file_set_integer (pr->priv->prefs_ini_file, "Preferences", key, value);
-	g_key_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
+	gmameui_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
 	
 	g_free (key);
 	
@@ -805,7 +805,7 @@ static void mame_gui_prefs_save_int_arr
 
 	/* Set the value and save the file */
 	g_key_file_set_integer_list (pr->priv->prefs_ini_file, "Preferences", key, value, NUMBER_COLUMN);
-	g_key_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
+	gmameui_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
 		
 	g_free (key);
 	
@@ -832,7 +832,7 @@ static void mame_gui_prefs_save_bool (Ma
 
 	/* Set the value and save the file */
 	g_key_file_set_boolean (pr->priv->prefs_ini_file, "Preferences", key, value);
-	g_key_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
+	gmameui_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
 		
 	g_free (key);
 	
@@ -859,7 +859,7 @@ static void mame_gui_prefs_save_string (
 
 	/* Set the value and save the file */
 	g_key_file_set_string (pr->priv->prefs_ini_file, "Preferences", key, value);
-	g_key_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
+	gmameui_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
 	
 	g_free (key);
 	
@@ -915,7 +915,7 @@ mame_gui_prefs_save_string_arr (MameGuiP
 		/* Set the value and save the file */
 		g_key_file_set_string_list (pr->priv->prefs_ini_file, "Preferences", key, value, n_va);
 	}
-	g_key_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
+	gmameui_file_save_to_file (pr->priv->prefs_ini_file, pr->priv->filename);
 		
 	GMAMEUI_DEBUG ("Saving string array %s... done", key);
 	
diff -Nurp gmameui.orig/src/io.c gmameui/src/io.c
--- gmameui.orig/src/io.c	2010-09-12 10:25:44.000000000 +0100
+++ gmameui/src/io.c	2014-10-13 01:05:36.265136265 +0100
@@ -124,7 +124,7 @@ my_dtostr (char * buf, gdouble d)
 
 /* Custom function to save the keyfile to disk. If the file does not exist,
    it will be created */
-gboolean g_key_file_save_to_file(GKeyFile *kf, const gchar *file)
+gboolean gmameui_file_save_to_file(GKeyFile *kf, const gchar *file)
 {
 	GError *error = NULL;
 	gsize length, nr;
diff -Nurp gmameui.orig/src/io.h gmameui/src/io.h
--- gmameui.orig/src/io.h	2009-10-26 06:20:07.000000000 +0000
+++ gmameui/src/io.h	2014-10-13 01:05:18.090923231 +0100
@@ -62,7 +62,7 @@ const gchar *
 my_dtostr(char* buf, gdouble d);
 
 gboolean
-g_key_file_save_to_file(GKeyFile *kf, const gchar *file);
+gmameui_file_save_to_file(GKeyFile *kf, const gchar *file);
 
 gboolean
 load_games_ini(void);
diff -Nurp gmameui.orig/src/mame_options.c gmameui/src/mame_options.c
--- gmameui.orig/src/mame_options.c	2009-09-22 11:39:28.000000000 +0100
+++ gmameui/src/mame_options.c	2014-10-13 01:06:40.174885397 +0100
@@ -492,7 +492,7 @@ mame_options_save_string (MameOptions *o
 	g_key_file_set_string (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 static void
@@ -519,7 +519,7 @@ mame_options_save_double (MameOptions *o
 	g_key_file_set_double (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 static void
@@ -546,7 +546,7 @@ mame_options_save_int (MameOptions *opts
 	g_key_file_set_integer (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 static void
@@ -573,7 +573,7 @@ mame_options_save_bool (MameOptions *opt
 	g_key_file_set_boolean (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 /* Gets the value from the keyfile for the specified key, which should be of the
diff -Nurp gmameui.orig/src/mame_options_legacy.c gmameui/src/mame_options_legacy.c
--- gmameui.orig/src/mame_options_legacy.c	2009-08-11 12:09:03.000000000 +0100
+++ gmameui/src/mame_options_legacy.c	2014-10-13 01:07:11.113248044 +0100
@@ -491,7 +491,7 @@ mame_options_legacy_save_string (MameOpt
 	g_key_file_set_string (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 static void
@@ -518,7 +518,7 @@ mame_options_legacy_save_double (MameOpt
 	g_key_file_set_double (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 static void
@@ -545,7 +545,7 @@ mame_options_legacy_save_int (MameOption
 	g_key_file_set_integer (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 static void
@@ -572,7 +572,7 @@ mame_options_legacy_save_bool (MameOptio
 	g_key_file_set_boolean (opts->priv->options_file, opt->category, opt->key, value);
 //	g_strfreev (stv);
 	
-	g_key_file_save_to_file (opts->priv->options_file, opts->priv->filename);
+	gmameui_file_save_to_file (opts->priv->options_file, opts->priv->filename);
 }
 
 /* Gets the value from the keyfile for the specified key, which should be of the
